client/components/Main.jsx

```diff
@@ -44,6 +48,7 @@ class Main extends Component {
 					</div>
 				<Route exact path='/products/:productId' component={Product} />
 				<Route path='/login' component={Login} />
+				<Route path='/history' component={ProductHistory} />
```

Comment:
Since order history pertains to specific user, the route should have userId in there.

Reply:
i know i didnt like /yourCart back when we did it cus it wasnt RESTful. But looking around on other online examples, when we make request a request, it should be to a restful route. In this case its just a display link, so we may be okay with it.

not to mention history/1, history/2 doesnt make much sense anyway.

How about we keep how I have it now, or we can change it to history/users/:id 

do you think thats better?